templates/admin/index.html

```diff
@@ -6,7 +6,7 @@
         <div class="col-sm-10 col-sm-offset-1">
             {% if not current_user.is_authenticated %}
             <p>
-                <a class="btn btn-primary" href="{{ url_for('security.login') }}">login</a> <a class="btn btn-default" href="{{ url_for('security.register') }}">register</a>
```

Comment:
change this to only be shown if `app.config["SECURITY_REGISTERABLE"] == True`

Reply:
added